cmd/csaf_checker/roliecheck.go

```diff
@@ -244,8 +253,8 @@ func (p *processor) processROLIEFeeds(feeds [][]csaf.Feed) error {
 		csaf.TLPLabelAmber,
 		csaf.TLPLabelRed,
 	} {
-		if _, ok := hasSummary[label]; !ok && len(p.labelChecker.advisories[label]) > 0 {
-			p.badROLIEfeed.warn(
+		if hasSummary.Contains(label) && len(p.labelChecker.advisories[label]) > 0 {
```

Comment:
The warning message gets triggered here - also if I check that against `sick.com` or another site, that has all their documents listed in a single ROLIE feed.